src/main.cpp

```diff
@@ -1,23 +1,24 @@
-#include <SFML/Graphics.hpp>
-#include <SFML/Window.hpp>
 #include <iostream>
+#include <wex/Engine.hpp>
 
-int main() {
-	sf::RenderWindow window(sf::VideoMode(800, 600), "SFML window");
-	window.setVerticalSyncEnabled(true);
-	window.setFramerateLimit(60);
+class MyApp final : public wex::Game {
+  private:
+	wex::Circle mCircle;
+
+  public:
+	void init() override {
+		mCircle = g->circle(100, 100, 30);
+		mCircle.setFillColor(sf::Color::Blue);
+	}
 
-	while (window.isOpen()) {
-		sf::Event event;
-		while (window.pollEvent(event)) {
-			switch (event.type) {
-			case sf::Event::Closed: window.close(); break;
-			case sf::Event::KeyPressed: std::cout << "ping\n"; break;
-			default: break;
-			}
-		}
-		window.clear();
-		window.display();
+	void update([[maybe_unused]] double dt) override {}
```

Comment:
In such cases you can just avoid naming the parameter altogether

Reply:
I kinda still want to indicate that it's delta time. I think this should be okay.